packages/protobuf/src/private/binary-format-common.ts

```diff
@@ -345,15 +367,21 @@ export function writeMapEntry(
 export function writeMessageField(
   writer: IBinaryWriter,
   options: BinaryWriteOptions,
-  type: MessageType,
-  fieldNo: number,
+  field: FieldInfo & { kind: "message" },
   value: any,
```

Comment:
Is this signature change not a compatibility issue? I guess I would have expected this to remain, for backwards-compatibility, and an alternate one introduced for group handling (with the shared logic factored into some shared function).

Reply:
Luckily, the function is not part of the public API. It is exported from this file, but it's not exported from the package.